### PR TITLE
Add resources needed to deploy rapidast-operator to smaug cluster

### PR DESCRIPTION
**Pull request by jpweiser**
Adds a catalog source, operator group, and subscription to deploy the RapiDAST operator to the smaug cluster.

### CODE REVIEW

File: cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: rapidast-catalog
+  namespace: rapidast
```
Comment (HumairAK) [inline cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml:6]: @jpweiser this namespace will also need to be added, you will need to add it to `base` much like the other manifests and then pulled in to the smaug overlay: 

namespace manifests are found here: 
https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces

you will need [to set a quota for the namespace](https://www.operate-first.cloud/apps/content/cluster-scope/add_resource_quotas.html).

Comment (jpweiser) [reply cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml:6]: This namespace was added previously https://github.com/operate-first/apps/tree/master/cluster-scope/base/core/namespaces/rapidast

I believe the quota is set [here](https://github.com/operate-first/apps/blob/master/cluster-scope/base/core/namespaces/rapidast/kustomization.yaml#L8)

**Comment by jpweiser** [comment]
/test kustomize-build

I think things are back up, so trying a retest

File: cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml (hunk 1 above)
Comment (jpweiser) [reply cluster-scope/base/operators.coreos.com/catalogsources/rapidast/catalogsource.yaml:6]: @HumairAK Does the above comment address your concerns, or did I miss something?

**Pull request merged**